mathesar_ui/src/component-library/tabs/TabContainer.svelte

```diff
@@ -19,10 +19,8 @@
   export let activeTab: Tab = tabs[0];
   export let idKey = 'id';
   export let labelKey = 'label';
-  export let linkKey = 'href';
   export let allowRemoval = false;
   export let preventDefault = false;
-  export let getLink: (arg0: unknown) => string;
```

Comment:
It assumes that each tab will have a link associated with it and will behave like an anchor tag. Since this component is not being used anywhere changing that behavior to make it more generic.